docs/painless/painless-contexts/painless-runtime-fields-context.asciidoc

```diff
@@ -7,6 +7,37 @@ Use a Painless script to calculate and emit
 See the {ref}/runtime.html[runtime fields] documentation for more information
 about how to use runtime fields.
 
+*Methods*
+
+`emit`::
+        Contains the fields to return values for based on the script valuation.
```

Comment:
`emit` "accepts" the values that the script produces. The parameter types are interesting - depending on the runtime field type you pass it different things.